brain/compartmentReportView.cpp

```diff
@@ -120,9 +56,23 @@ const CompartmentReportMapping& CompartmentReportView::getMapping() const
 }
 
 std::future<CompartmentReportFrame> CompartmentReportView::load(
-    const float timestamp)
+    double timestamp)
 {
+    const double start = _impl->report->getStartTime();
+    const double end = _impl->report->getEndTime();
+
+    if (timestamp < start || timestamp > end)
```

Comment:
if timestamp == end it should also be considered invalid because the frame time window interval is open on the right.